test/tests/functional/pbs_user_reliability.py

```diff
@@ -109,11 +109,16 @@ def test_create_resv_from_job_using_qsub(self):
             s_nodect_before = '0'
             s_ncpus_before = '0'
 
+        now = time.time()
+
         a = {ATTR_W: 'create_resv_from_job=1'}
         job = Job(TEST_USER, a)
         jid = self.server.submit(job)
         self.server.expect(JOB, {ATTR_state: 'R'}, jid)
 
+        self.server.log_match("Reject reply code=15095", starttime=now,
+                              interval=2, max_attempts=20, existence=False)
+
```

Comment:
This is a forced 40s wait in the test.  Is this what you want?  Maybe max_attempts can be 10?